test/lolex-test.js

```diff
@@ -1687,95 +1687,58 @@ describe("lolex", function () {
             assert.equals(fakeDate.getTime(), date.getTime());
         });
 
-        it("returns regular date when calling with timestamp", function () {
-            var date = new Date();
-            var fakeDate = this.clock.Date(date.getTime());
```

Comment:
in the cases of these deleted tests, maybe just change `this.clock.Date(...)` to `new this.clock.Date(...)`? Unless the exact same tests are somewhere else as well